WMCDuplicateRemover.Tests/EPG/EPGIntegrationTests.cs

```diff
@@ -17,7 +13,9 @@ public class EPGIntegrationTests
         [TestFixtureSetUp]
         public void SetUp()
         {
-            tv = new XmlTvEpgWrapper(@"..\..\EPG\TestData\EPG.xml");
+            var dateTimeMock = new Mock<IDateTime>();
```

Comment:
An interface is a contract. Any object that implements an interface is contractually obligated to implement what's specified in the interface. In this case my interface specifies that there is a method named Now that returns a type of DateTime. This allows Moq to figure out that the object it generates HAS to have a method called now on it that returns a DateTime. On line 17 Moq allows me to set what the Now method actually returns. I cannot think of a case I'd use a Mock that wasn't in a test case. Doesn't mean someone hasn't found a reason to but I can't think of it.